src/main/java/mate/academy/spring/validator/EmailValidator.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.validator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
```

Comment:
https://github.com/mate-academy/jv-spring-rest-web-practice#:~:text=!%20Don%27t%20create%20GlobalExceptionHandler%2C%20custom%20validation%20annotations%20in%20this%20HW%20(this%20is%20not%20specified%20in%20the%20task%20condition)